inc/policydeployapplication.class.php

```diff
@@ -118,7 +118,8 @@ public function unicityCheck($value, $itemtype, $itemId, PluginFlyvemdmFleet $fl
       $fleetId = $fleet->getID();
       $task = new PluginFlyvemdmTask();
       $rows = $task->find("`plugin_flyvemdm_fleets_id` = '$fleetId'
-            AND `itemtype` = '$itemtype' AND `items_id` = '$itemId'", '', '1');
+            AND `plugin_flyvemdm_policies_id` = '" . $this->policyData->getID() . "' 
+            AND `items_id` = '$itemId'", '', '1');
```

Comment:
What was the problem by checking the itemtype instead if the policy id ?

Reply:
The policies of  deploy and removal app now saves the class name string so to identify them I'm searching for the policy id.

Reply:
Ok for this; I think this also fixes a bug.